tools/pdf2txt.py

```diff
@@ -309,6 +317,9 @@ def parse_args(args: Optional[List[str]]) -> argparse.Namespace:
             if parsed_args.outfile.endswith(override):
                 parsed_args.output_type = alttype
 
+    if parsed_args.ignore_unmapped:
+        PDFLayoutAnalyzer.handle_undefined_char = lambda *args: ""  # type: ignore
```

Comment:
This increases technical debt. I prefer to reduce tech debt and fix bugs over adding features. 

I would like to understand more about why using `LAParams` is not a good option for you. In my understanding, `LAParams` is the way to communicate parameters from the command line to the `PDFLayoutAnalyzer`.

Reply:
Indeed, this is a hack... maybe there should be a better way to communicate "don't do layout analysis" to `PDFLayoutAnalyzer`, see my reply to your other comment.